src/kartograf/filters/ring_changes.py

```diff
@@ -162,3 +163,55 @@ def get_atom_ring_hybridization_map(rdmol: Chem.Mol) -> dict[int, bool]:
             filtered_mapping[ai] = aj
 
     return filtered_mapping
+
+def filter_fused_ring_changes(molA: Chem.Mol, molB: Chem.Mol, mapping: ATOM_MAPPING) -> ATOM_MAPPING:
+    """
+    Remove cases where a fused ring is partially mapped and could be considered broken
+    see <https://github.com/OpenFreeEnergy/kartograf/pull/56> for more details.
```

Comment:
I know the other docstrings aren't great, but is there more detail we can provide?

(mostly thinking that from a user perspective it might be confusing now that we have this and `filter_wole_ring`, etc..)

Maybe it's not the best place for this.. but I don't know that we have a userguide in kartograf.

Reply:
Okay tried to add some more detail but I don't think many users will see this here, I'll add a section to the tutorial on this option using the molecule in the issue as an example.